blockstore/splitstore/splitstore.go

```diff
@@ -1063,6 +1119,7 @@ func (s *SplitStore) doCompact(curTs *types.TipSet) error {
 	}
 	log.Infow("sorting done", "took", time.Since(startSort))
 
+	// Can we add a comment explaining why we might want to do this here instead of just waiting till we purge?
```

Comment:
sure; basically I want to minimize the invocations inside purge, but we can also ellide it.

Reply:
the only reason I am doing it seperately is there is likely a large number of references accumulated during sort (it's slow!) and I didn't want to do that in purge context.

Reply:
added comment.